newlib/ChangeLog.ARC

```diff
@@ -1,3 +1,8 @@
+2015-05-28 Yuriy Kolerov <yuriy.kolerov@synopsys.com>
```

Comment:
There should be two whitespaces between date and name, and between name and email:
```
2015-05-28  Yuriy Kolerov  <yuriy.kolerov@synopsys.com>
```